frontend/src/environments/environment.prod.ts

```diff
@@ -4,4 +4,5 @@ export const environment = {
   production: true,
     assets: '/static/assets',
     version,
+    apiUrl: '/api'
```

Comment:
Just making sure: `environment.ts` (dev) has a trailing slash here. We probably want them to be consistent.

Reply:
Good catch!